Formula/luneta.rb

```diff
@@ -0,0 +1,19 @@
+class Luneta < Formula
+  desc "Command-line fuzzy finder"
+  homepage "https://github.com/fbeline/luneta/"
+  url "https://github.com/fbeline/luneta/archive/v0.7.2.tar.gz"
+  sha256 "e84956500c6ccac63b0cc2969d82cad09ab8689a304c571b783f4244aeee5b61"
+
+  depends_on "dub" => :build
+  depends_on "ldc" => :build
+  depends_on "ncurses"
```

Comment:
Build error:
`Dependency 'ncurses' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.`

Some Linux distribution does not come with `ncurses` pre-installed. If change it to `uses_from_macos`, it may break in Linux right?

Reply:
No, because "uses_from_macos" means "download_on_linux". In addition to that, linux support is not a concern for this repo.